GameEngine/GameEngine/Engine/Utils/ThreadName.hpp

```diff
@@ -2,6 +2,7 @@
 
 #ifdef WIN32
 
+#define NOMINMAX
```

Comment:
I dont know why but when I updated glm, the only problem I had was that glm::min and glm::max where in conflict with the min and max functions from windows (it shouldn't as it is in the glm namespace...) so this define just removes the min and max functions from windows.h (we dont use them anyway).